### PR TITLE
London - Class - 7 - Tanika - Javascript - Week - 1

### PR DESCRIPTION
**Pull request by tanikaDev**
completed exercises and mandatory tasks for javascript week1

# Your Details

- Your Name: Tanika 
- Your City: London
- Your Slack Name:  Tanika Robinson

# Homework Details

- Module: Javascript
- Week: 1

### CODE REVIEW

File: exercises/K-functions-parameters/exercise3.js [hunk 1]
```diff
@@ -1,4 +1,7 @@
 // Write your function here
+function createGreeting(name){
+    return "Hi " + name + ", how are you?"
```
Comment (ybayraktutan) [inline exercises/K-functions-parameters/exercise3.js:3]: Needs a semi-colon ; at the end of the line of code

**Review by ybayraktutan — COMMENTED** [review]
Great work Tanika! I have added a few comments :)

File: exercises/F-strings-methods/exercise.js [hunk 2]
```diff
@@ -1,3 +1,7 @@
 // Start by creating a variable `message`
 
+const person = "Tanika";
+const personLength = person.length;
+var message = `My name is ${person} and my name is ${personLength} characters long`
```
[the remark this answers is not in the document]
Comment (Trobinsonshoots) [reply exercises/F-strings-methods/exercise.js]: Added it now, same with the other two. Thanks

File: exercises/G-numbers/exercise.js [hunk 3]
```diff
@@ -1 +1,6 @@
 // Start by creating a variables `numberOfStudents` and `numberOfMentors`
+var students = 15;
+var mentors = 8;
+var message = "Total number of students and mentors: " + (students+mentors);
+
+console.log(message)
```
[the remark this answers is not in the document]
Comment (Trobinsonshoots) [reply exercises/G-numbers/exercise.js]: Added it now, and to the other two. Thanks

**Pull request closed without merging**